addons/l10n_ae/views/report_invoice_templates.xml

```diff
@@ -3,31 +3,117 @@
     <template id="report_invoice_document" inherit_id="account.report_invoice_document">
         <xpath expr="//p[@name='payment_term']" position="after">
             <p t-if="o.company_id.country_id.code == 'AE' and o.partner_id.country_id.code != 'AE' and o.env.ref('l10n_ae.gcc_countries_group') in o.partner_id.country_id.country_group_ids">
-                Supply between <b>United Arad Emirates</b> and <b><span t-field="o.partner_id.country_id.name"/></b>
+                Supply between <b>United Arad Emirates</b> and
+                <b>
+                    <span t-field="o.partner_id.country_id.name"/>
+                </b>
             </p>
         </xpath>
         <xpath expr="//h2/span" position="before">
-            <span t-if="o.company_id.country_id.code == 'AE' and o.move_type in ['out_invoice', 'out_refund']">Tax</span>
+            <span t-if="o.company_id.country_id.code == 'AE' and o.move_type in ['out_invoice', 'out_refund']">TAX
+            </span>
         </xpath>
 
+        <xpath expr="//thead//th[@name='th_taxes']" position="replace">
+            <th name="th_taxes"
+                t-attf-class="text-left {{ 'd-none d-md-table-cell' if report_type == 'html' else '' }}">
+                <span t-if="o.company_id.country_id.code == 'AE'">VAT</span>
+                <span t-else="">Taxes</span>
+            </th>
+            <th t-if="o.company_id.country_id.code == 'AE'" name="tax_amount"
+                t-attf-class="text-left {{ 'd-none d-md-table-cell' if report_type == 'html' else '' }}">
+                <span>VAT Amount</span>
+            </th>
+        </xpath>
+
+        <xpath expr="//tbody" position="replace">
```

Comment:
Replace is often a really bad idea; especially for such a big element. It means any inheritance done by previous modules (maybe sales/purchases/hr related) will be erased completely.